include/storage/io.hpp

```diff
@@ -108,6 +108,13 @@ class FileReader
         }
     }
 
+    void ReadInto(std::string &target)
```

Comment:
We want to keep these classes as small as possible, hence I would suggest adding a function to `storage/io.hpp` like `read(FileReader, std::string)` and `write(FileWrite, const std::string &)`.